### ISSUE TITLE
Blank Screen

### ISSUE THREAD

**Issue by AKA-Syenite** [manga-download/hakuneko#498]
![image](https://user-images.githubusercontent.com/918621/66272813-3d8f6400-e83c-11e9-8077-5b1eecdc3eb7.png)

When launching, it checks for updates, then gets stuck on a blank screen. On 64-bit Windows 10, tried nightly-20191006.1 & nightly-20191005.2.

**Comment by AKA-Syenite** [comment]
Output when started from console

```
> --------------
> Framework Info
> --------------
> Electron : 6.0.10
> Chrome   : 76.0.3809.146
> Node     : 12.4.0
> 
> 
> -------------
> Configuration
> -------------
> Update URL           : https://manga-download.github.io/hakuneko/6.0/latest
> Connector Protocol   : connector
> Application Protocol : hakuneko
> Startup URL          : hakuneko://cache/index.html
> AppCache Directory   : D:\Downloads\hakuneko-desktop_6.0.10_windows-portable_amd64\cache
> UserData Directory   : D:\Downloads\hakuneko-desktop_6.0.10_windows-portable_amd64\userdata
> 
>  2019-10-06T17:31:01.104Z <ERROR>    Update Failed! Error: Status: 301
>     at ClientRequest.<anonymous> (D:\Downloads\hakuneko-desktop_6.0.10_windows-portable_amd64\resources\app.asar\UpdateServerManager.js:50:28)
>     at Object.onceWrapper (events.js:288:20)
>     at ClientRequest.emit (events.js:200:13)
>     at HTTPParser.parserOnIncomingClient [as onIncoming] (_http_client.js:565:23)
>     at HTTPParser.parserOnHeadersComplete (_http_common.js:116:17)
>     at TLSSocket.socketOnData (_http_client.js:452:22)
>     at TLSSocket.emit (events.js:200:13)
>     at addChunk (_stream_readable.js:294:12)
>     at readableAddChunk (_stream_readable.js:275:11)
>     at TLSSocket.Readable.push (_stream_readable.js:210:10)
```

**Comment by 27136773** [comment]
Related #13

**Comment by 27136773** [comment]
This seems to be a new issue, which is unrelated to #13, the problem is basically that the new client uses HTTPS for its GitHub update url, which is always redirected to HTTP.

**Comment by 27136773** [comment]
Has been fixed in 5ceb1f72
